src/base/cpp/config/discover.ml

```diff
@@ -23,23 +23,39 @@ let () =
           else []
         else []
       in
-
       (*
-          Find the absolute path to vcpkg's library.
+          Find the absolute path to vcpkg's libraries & headers.
           Unfortunately, %{project_root} in the dune file always returns a relative path and in order
           for -L to work properly, we need to give an absolute path.
-          The 'pwd' below returns a sub-directory of the _build directory so currently, we rely
-          on the 'realpath' utility & and the build directory location.
+          We rely on a simple python program (in vcpkg_cmd) to do so portably.
       *)
-      let ic = Unix.open_process_in "pwd" in
-      let cmd =
-        "realpath " ^ input_line ic
-        ^ "/../../../../../vcpkg_installed/x64-linux-dynamic/lib"
+      let vcpkg_cmd =
+        "python3 -c 'import os, sys; print(os.path.realpath(\"../../../../../"
```

Comment:
Can we use [`realpath (1)`](https://man7.org/linux/man-pages/man1/realpath.1.html) instead?

Reply:
Originally I was actually, but this doesn't exist on OS X by default (and homebrew didn't find it). So a portable way (why is better, I'd say) is this simple solution.

Reply:
OK.

The better solution would be to use `Unix.realpath`, but it is available since OCaml 4.13. The minimal supported version for Scilla is 4.12, so we cannot use it at the moment.